Assets/VRMPAssets/Scripts/Player/PlayerAppearanceMenu.cs

```diff
@@ -6,17 +6,29 @@
 namespace XRMultiplayer
 {
     /// <summary>
-    /// A simple example of how to setup a player appearance menu and utilize the bindable variables.
+    /// A simple example of how to setup a player appearance menu,
+    /// utilize the bindable variables, AND use PlayerPrefs for persistence.
     /// </summary>
     public class PlayerAppearanceMenu : MonoBehaviour
     {
+        [Header("Color Setup")]
         [SerializeField] Color[] m_PlayerColors;
-        [SerializeField] TMP_InputField m_PlayerNameInputField;
         [SerializeField] Image m_PlayerIconColor;
 
+        [Header("Name Field")]
+        [SerializeField] TMP_InputField m_PlayerNameInputField;
+
+        // PlayerPrefs keys
+        private const string USER_NAME_KEY = "UserName";
+        // Example key if an avatar dropdown is added later:
+        private const string AVATAR_INDEX_KEY = "AvatarIndex";
 
         void Awake()
         {
+            string savedName = PlayerPrefs.GetString(USER_NAME_KEY, "Player");
+
+            XRINetworkGameManager.LocalPlayerName.Value = savedName;
```

Comment:
You can use `XRINetworkGameManager.LocalPlayerName.Value = PlayerPrefs.GetString(USER_NAME_KEY, "Player");`, the local var `savedName` is unnecessary.